app/controllers/DataController.scala

```diff
@@ -9,9 +9,10 @@ import javax.inject.{Inject, Singleton}
 class DataController @Inject() (dataService: DataService, cc: ControllerComponents) extends AbstractController(cc) with play.api.i18n.I18nSupport {
 
   def submissions(name: Option[String] = None)  = Action { implicit request: Request[AnyContent] =>
-    val query = request.getQueryString(key="name")
-    val queryResult = dataService.search(query)
+    val queryName = request.getQueryString(key="name")
```

Comment:
its possible to define the query params in the app.routes along with default values
`submissions(name: Option[String] ?= None, age: Option[Int] ?= None)` etc